### PR TITLE
Auth component

### PR DESCRIPTION
**Pull request by raonismaneoto**

### CODE REVIEW

File: frontend/src/components/Auth.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    });
+  };
+
+  render() {
+    const loginFlag = this.state.login;
+    const loginBg = this.logInBackGround();
+    const singUpBg = this.signUpBackGround();
+    
+    return (
+      <div style={{'height': '200px', 'width': '300px'}}>
```
Comment (gugabribeiro) [inline frontend/src/components/Auth.js]: Extract the code of style property into index.css and refence it as class name property in this div.

File: frontend/src/components/Auth.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        <article className="ba b--purple content-center h-100">
+          <div className="flex justify-center h-20" style={{'backgroundColor': "#FFFFFF"}}>
+            <div className="h-100 w-50 tc">
```
Comment (gugabribeiro) [inline frontend/src/components/Auth.js]: Same here.

File: frontend/src/components/Auth.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+              <div style={{'height': '40px', 'backgroundColor': loginBg}}
```
Comment (gugabribeiro) [inline frontend/src/components/Auth.js]: Same here.

File: frontend/src/components/Auth.js [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+                onClick={this.changeState}>
+                <span style={{'fontSize': '25px', 'color': loginFlag? '#A463F2' : '#FFFFFF'}}>
```
Comment (gugabribeiro) [inline frontend/src/components/Auth.js]: Same here.

File: frontend/src/components/Auth.js [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+                  Log In
+                </span>
+              </div>
+            </div>
+            <div className="h-100 w-50 tc">
+              <div style={{'height': '40px', 'backgroundColor': singUpBg}}
```
Comment (gugabribeiro) [inline frontend/src/components/Auth.js]: Same here.

File: frontend/src/components/Auth.js [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+                onClick={this.changeState}>
+                <span style={{'fontSize': '25px', 'color': loginFlag? '#FFFFFF' : '#A463F2'}}>Sign Up</span>
```
Comment (gugabribeiro) [inline frontend/src/components/Auth.js]: Same here.

File: frontend/src/components/Auth.js [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+              </div>
+            </div>
+          </div>
+          <div className="flex justify-center h-40">
+            <form className="flex flex-column pa4 black-80">
+              <div className="measure" style={{'borderRadius': '50px'}}>
```
Comment (gugabribeiro) [inline frontend/src/components/Auth.js]: There is a tachyons class that deals with rounded borders. It is br-X, where X is how much rounded should it be, see reference for more details.

File: frontend/src/components/Auth.js [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+                <input id="username" className="input-reset ba b--black-20 mb2 db w-100" 
+                  type="text" aria-describedby="name-desc" style={{'borderRadius': '50px'}} onChange={this.setCredentials('email')}/>
```
Comment (gugabribeiro) [inline frontend/src/components/Auth.js]: Same of the above comment.

File: frontend/src/components/Auth.js [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+              </div>
+              <div className="measure">
+                <input id="name" className="input-reset ba b--black-20 mb2 db w-100" 
+                  type="password" aria-describedby="name-desc" style={{'borderRadius': '50px'}} onChange={this.setCredentials('password')}/>
```
Comment (gugabribeiro) [inline frontend/src/components/Auth.js]: Same of the above comment.

File: frontend/src/components/Auth.js [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+              </div>
+            </form>
+          </div>
+          {this.button()}
+        </article>
+      </div>
+    );
+  }
```
Comment (lucasaarcoverde) [inline frontend/src/components/Auth.js]: Missing semicolon

File: frontend/src/components/MainPage.js [hunk 11]
```diff
@@ -1,10 +1,12 @@
...
 
 export default class MainPage extends Component {
   render() {
     return (
       <div className="ma2">
         <h1>tutO(r)</h1>
+        <Auth />
       </div>
     );
   }
```
Comment (lucasaarcoverde) [inline frontend/src/components/MainPage.js]: Missing semicolon

**Pull request merged**